docs/blogs/posts/2024/2024-01-parakeet.md

```diff
@@ -0,0 +1,102 @@
+
+## Architecture Details
+
+Parakeet models are based on the [Fast Conformer architecture published in ASRU 2023](https://arxiv.org/abs/2305.05084). Fast Conformer is an optimized version of the Conformer model with 8x depthwise-separable convolutional downsampling, modified convolution kernel size, and an efficient subsampling module. Additionally it supports inference on very long audio segments (up to 11 hours of speech) on an A100 80GB card using Local Attention. The model is trained end-to-end using the Transducer decoder (RNNT) or Connectionist Temporal Classification decoder. For further details on long audio inference, please refer to the ICASSP 2024 paper “[Investigating End-to-End ASR Architectures for Long Form Audio Transcription](https://arxiv.org/abs/2309.09950)”.
+
+<figure markdown>
+  ![Parakeet architecture](https://github.com/NVIDIA/NeMo/releases/download/v1.21.0/asset-post-2024-01-03-parakeet_arch.png)
+  <figcaption><b>Figure 1.</b> <i> Fast Conformer Architecture shows blocks of downsampling, conformer encoder blocks with limited context attention (LCA), and global token (GT).</i></figcaption>
+</figure>
+
+## Usage
+
+NVIDIA NeMo can be installed as a pip package as shown below. Cython and PyTorch (2.0 and above) should be installed before attempting to install NeMo Toolkit.
+
+Then simply use:
+```bash 
+pip install nemo_toolkit['asr']
+```
+
+Once installed, you can evaluate a list of audio files as follows:
+```python
+import nemo.collections.asr as nemo_asr
+asr_model = nemo_asr.models.ASRModel.from_pretrained(model_name="nvidia/parakeet-rnnt-1.1b")
+transcript = asr_model.transcribe(["some_audio_file.wav"])
+```
+
+## Long-Form Speech Inference
+
+Once you have a Fast Conformer model loaded, you can easily modify the attention type to limited context attention after building the model. You can also apply audio chunking for the subsampling module to perform inference on huge audio files!
+
+!!! note
+    These models were trained with global attention, and switching to local attention will degrade their performance. However, they will still be able to transcribe long audio files reasonably well.
+
+For limited context attention on huge files (upto 11 hours on an A100), perform the following steps:
+
+```python
+# Enable local attention
+asr_model.change_attention_model("rel_pos_local_attn", [128, 128])  # local attn
+
+# Enable chunking for subsampling module
+asr_model.change_subsampling_conv_chunking_factor(1)  # 1 = auto select
+
+# Transcribe a huge audio file
+asr_model.transcribe(["<path to a huge audio file>.wav"])  # 10+ hours !
+```
+
+## Additional Resources
+
+* [HuggingFace ASR Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard)
```

Comment:
Might be good to also link the evaluation codebase used by the leaderboard for NeMo somewhere as well?